language.json

```diff
@@ -7,6 +7,18 @@
         "loggedIn": "Logged in as %{name}",
         "logout": "Logout",
         "collection": "Collection",
+        "collections": {
+          "Alger": "Algae",
```

Comment:
This is the exact reason why i did not do it like this. This is an antipattern. You can remove the Norwegian translation if it doesn't do anything with the keystring. E.g. If it doesn't remove the spaces.